include/ninshiki_cpp/utils/contours.hpp

```diff
@@ -37,12 +37,12 @@ class Contours
   explicit Contours(std::vector<std::vector<cv::Point>> contours);
   explicit Contours(cv::Mat binary_mat);
 
+  std::vector<std::vector<cv::Point>> contours;
+
   cv::Mat get_binary_mat(cv::Size mat_size);
   cv::Mat get_binary_mat_line(cv::Size mat_size, int line_size);
 
   std::vector<std::vector<cv::Point>> get_contours();
-  void set_contours(std::vector<std::vector<cv::Point>> contours);
-  void add_contour(std::vector<cv::Point> contour);
```

Comment:
The `get_contours` function is unnecessary since the `contours` variable is already public.